packages/subscriber/src/components/add-form/style.scss

```diff
@@ -55,6 +55,17 @@
 	.components-notice.is-info {
 		background: var(--studio-gray-0);
 	}
+
+	.add-subscriber__form-skip-link-wrapper {
+		display: flex;
+		justify-content: flex-start;
+		padding: 16px 0;
```

Comment:
Please use rem unit instead of pixel. `16px` => `1rem`

Reply:
Updated. And thanks for taking the time to review @bogiii !